styles.css

```diff
@@ -1,543 +1,543 @@
-    }
-    .css3{
-      width: 75vw;
-      height: 15vw;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-      font-weight: 500;
-      font-size: 4vw;
-      line-height: 5vw;
-      display: flex;
-      align-items: center;
-      color: #253858;
-      text-indent: 18vw;
-      padding-top:  2vw;
-      background: url("css3.png");
-      background-size: 12vw 12vw;
-      background-repeat: no-repeat;
-      background-position: 4vw 2vw;
-      border: 1px solid #FFFFFF;
-    }
-    .line{
-      width: 78.67vw;
-      height: 1vw;
-      margin-left: 1vw;
-      margin-right: 1vw;
-      border: #DFE1E6;
-    }
\ No newline at end of file
+body{
+  width: 375px;
+  height: 100%;
+  overflow-x: hidden;
+}
+main{
+  width: 100%;
+  min-height: 100vh;
+  background-image: url("background.png");
+  background-size: 95vw 23vw;
+  background-repeat: no-repeat;
+}
+.navgation {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  max-width: 100%; 
+  max-height: 15vw;
+  align-items: center;
+}
+.menu {
+  max-width: 50vw;
+  max-height: 10vw;
+  margin-left: 68.5vw;
+  margin-right: 2vw;
+  align-items: center;
+}
+.para {
+  max-width: 100vw;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 5.5vw;
+  line-height: 6vw;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+.feyisa {
+  max-width: 100vw;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 13.7vw; 
+  line-height: 13vw;
+  display: flex;
+  align-items: center;
+  color: #172B4D;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  top: 25%;
+}
+.let {
+  width: 35vw;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 4.267vw;
+  line-height: 5vw;
+  color: #7F8CFF;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 3vw;
+}
+.part{
+  width: 100vm;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin-left: 0.5vw;
+  margin-right: 1vw;
+  }
+.icon{
+display: flex;
+flex-grow: 1;
+max-width: 40vw;
+max-height: 25vm;
+margin-top: 5vm;
+justify-content: space-between;
+gap: 5vm;
+}
+.icon-1{
+  display: flex;
+  flex-grow: 1;
+  max-width: 100vw;
+  min-height: 40vh;
+  background-image: url("bottomback.png");
+  background-size: 100vw 20vw;
+  background-repeat: no-repeat;
+  }
+.logo {
+  max-width: 40vw;
+  font-family:'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 5vw;
+  line-height: 5vw;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070FF;
+  flex: none;
+  order: 0;
+  margin-left: 1vw;
+}
+.bottom{
+  max-width: 5vw;
+  margin-left: 85vw;
+  margin-top: 0vw;
+}
+.page{
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  margin-left: 3vw;
+  margin-right: 5vw;
+  gap: 10vw;
+  margin-left: 2vw;
+  margin-right: 2vw;
+  background: #F7F7F9;
+  overflow-x: hidden;
+}

```

Comment:
- [ ] The main container of the projects has a display: grid. i highly recommend you use a grid system to display